### PR TITLE
CIV-14430 Dashboard notification for GA Lip Hwf

### PR DESCRIPTION
**Pull request by kenneth-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-14430

### Change description ###
Dashboard Notification - GA for LiPs

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ga-ccd-definition/CaseEvent/Camunda/DashboardEventsGAspec-CUI-nonprod.json [hunk 1]
```diff
@@ -102,5 +102,18 @@
     "ShowSummary": "N",
     "ShowEventNotes": "N",
     "RetriesTimeoutURLAboutToSubmitEvent": 0
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "ID": "APPLICANT_LIP_HWF_DASHBOARD_NOTIFICATION",
+    "Name": "respondent notification",
```
Comment (neeta-hmcts) [inline ga-ccd-definition/CaseEvent/Camunda/DashboardEventsGAspec-CUI-nonprod.json:109]: applicant

File: ga-ccd-definition/CaseEvent/Camunda/DashboardEventsGAspec-CUI-nonprod.json [hunk 2]
```diff
@@ -102,5 +102,18 @@
...
+    "Description": "respondent notification",
```
Comment (neeta-hmcts) [inline ga-ccd-definition/CaseEvent/Camunda/DashboardEventsGAspec-CUI-nonprod.json:110]: applicant

File: ga-ccd-definition/CaseEvent/Camunda/DashboardEventsGAspec-CUI-nonprod.json (hunk 1 above)
Comment (kenneth-hmcts) [reply ga-ccd-definition/CaseEvent/Camunda/DashboardEventsGAspec-CUI-nonprod.json]: Updated for the changes

File: ga-ccd-definition/CaseEvent/Camunda/DashboardEventsGAspec-CUI-nonprod.json (hunk 2 above)
Comment (kenneth-hmcts) [reply ga-ccd-definition/CaseEvent/Camunda/DashboardEventsGAspec-CUI-nonprod.json]: Updated for the changes

**Review by neeta-hmcts — DISMISSED** [review]

**Review by deepthidoppalapudihmcts — DISMISSED** [review]

**Review by karthick-mohanasundaram-HMCTS — APPROVED** [review]

**Review by sankhajuria — APPROVED** [review]

**Pull request merged**